kordac/tests/assets/conditional/doc_example_override_html_expected.html

```diff
@@ -0,0 +1,14 @@
+<script>
+ if (version == 'teacher') {
+  return "<p>This is text that only teachers should see.</p>
+";
```

Comment:
why are the end quotes on a newline, is this intentional? Github's highlighting of this burns my eyes